src/toICN-after.js

```diff
@@ -53,24 +61,42 @@ else{
   keyMinorSignature = resultKeyMinorSignature;
 }
 //表示書き換え関係
-chordElms.forEach((e) => {
-  let icn = exports.toICN(""+e.firstChild.nodeValue);
-  let isSharp = false;
-  let isSwap = false;
-  let isBlueChord = false;
-  //シャープ、スワップ、特定のセブンスコード等の条件を満たすかどうかを調べる
-  if(icn!=""){
-    e.firstChild.nodeValue = icn;
-    if(icn.includes("#")){isSharp = true;}
-    if(icn.includes("~")){isSwap = true;}
-    if("1[7],1#[7],4[7],4#[7],2[M7],2#[M7],3[M7],5[M7],5#[M7],6[M7],6#[M7],7[M7]".split(",").includes(icn) || /\[sus4\]|\[aug\]|\[dim\]|\[m7\-5\]$/.test(icn)){
-      isBlueChord = true;
+(isKeysOnChordElms?keyChordElms:chordElms).forEach((e) => {
```

Comment:
```suggestion
(keyChordElms.length != 0?keyChordElms:chordElms).forEach((e) => {
```

として isKeysOnChordElmsフラグ自体を削除するのはどうでしょう？ L65の判定からも削除して問題ないように見えます。